test/integration/client-side-operations-timeout/client_side_operations_timeout.unit.test.ts

```diff
@@ -0,0 +1,50 @@
+/* eslint-disable @typescript-eslint/no-empty-function */
+/**
+ * The following tests are described in CSOTs spec prose tests as "unit" tests
+ * The tests enumerated in this section could not be expressed in either spec or prose format.
+ * Drivers SHOULD implement these if it is possible to do so using the driver's existing test infrastructure.
+ */
+
+describe.skip('CSOT spec unit tests', () => {
+  context('Operations should ignore waitQueueTimeoutMS if timeoutMS is also set.', () => {});
+
+  context(
+    'If timeoutMS is set for an operation, the remaining timeoutMS value should apply to connection checkout after a server has been selected.',
+    () => {}
+  );
+
+  context(
+    'If timeoutMS is not set for an operation, waitQueueTimeoutMS should apply to connection checkout after a server has been selected.',
+    () => {}
+  );
+
+  context(
+    'If a new connection is required to execute an operation, min(remaining computedServerSelectionTimeout, connectTimeoutMS) should apply to socket establishment.',
+    () => {}
+  );
+
+  context(
+    'For drivers that have control over OCSP behavior, min(remaining computedServerSelectionTimeout, 5 seconds) should apply to HTTP requests against OCSP responders.',
+    () => {}
+  );
+
+  context(
+    'If timeoutMS is unset, operations fail after two non-consecutive socket timeouts.',
```

Comment:
Is this timeoutMS for a singular operation, or multiple? 

If it's timeoutMS for a singular operation, I suggest we move it into a block with the other unset TimeoutMS for neatness.

Otherwise LGTM!

Reply:
Sounds good